docs/articles/advanced_topics/generic_host.md

```diff
@@ -17,37 +17,34 @@ To get started, you'll need to write some code that configures and runs your app
 private static async Task Main()
 {
     await Host.CreateDefaultBuilder()
-      .UseConsoleLifetime()
-      .ConfigureServices((hostContext, services) =>
-      {
-          services.AddHostedService<YourBotHost>();
-      })
-      .RunConsoleAsync();
```

Comment:
Probably should've been mroe specific in that I was referring to the indentation, sorry. I'm a clown.

![:cat_clown:](https://cdn.discordapp.com/emojis/907330621074993253.webp?size=48&quality=lossless)